robolectric/src/test/java/org/robolectric/shadows/ShadowBluetoothDeviceTest.java

```diff
@@ -297,7 +297,7 @@ public void getSetAlias() {
       Method getAliasName = android.bluetooth.BluetoothDevice.class.getMethod("getAlias");
       assertThat((String) getAliasName.invoke(device)).isEqualTo(aliasName);
     } catch (ReflectiveOperationException e) {
-      throw new AssertionError("Failure accessing getAlias via reflection", e);
+      throw new LinkageError("Failure accessing getAlias via reflection", e);
```

Comment:
In test scenario, the `AssertionError` is more clear than `LinkageError`. We just want to throw an `AssertionError` to notify this test's assertion failed, caused by `ReflectiveOperationExpcetion`.  What about suppressing it?

Reply:
ahh okay 👍🏻

Reply:
https://github.com/robolectric/robolectric/pull/7400/files#diff-de45bbc32003dba3c1e3a4e5e24aa0e25a0bc18d2e96480919285a10cd8e2209R290